src/display/optional_content_config.js

```diff
@@ -207,6 +207,10 @@ class OptionalContentConfig {
     this.#cachedGetHash = null;
   }
 
+  get isEmpty() {
+    return this.#initialHash === null;
+  }
+
   get hasInitialVisibility() {
     return this.getHash() === this.#initialHash;
```

Comment:
This seems like a pre-existing bug in the optional-content implementation, so can we please adjust the existing getter to handle this instead?
```suggestion
  get hasInitialVisibility() {
    return this.#initialHash === null || this.getHash() === this.#initialHash;
```

Reply:
It was my initial patch and I changed it to add isEmpty in order to avoid to break something.